examples/common/mod.rs

```diff
@@ -73,7 +99,98 @@ impl Default for PacketStats {
         PacketStats {
             start_time: std::time::Instant::now(),
             current: 0,
-            history: heapless::HistoryBuffer::new(),
+            bytes: 0,
+            message_history: History::new(),
+            byte_history: History::new(),
+        }
+    }
+}
+
+fn log_metric(metric_name: &str, metric_value: u128, elapsed_micros: u128, history: &History) {
+    log::info!(
+        "{}: {:>8} | 1s average: {:>8}/s) | {}s average: {:>8}/s{}",
```

Comment:
```
"{:>9}: {:>9} | 1s average: {:>9}/s | {}s average: {:>9}/s{}"
```

ie

- Use `>9` for the metric name so that the rest of the line is aligned for all metrics. (9 is long enough for `"messages"` and `"bytes"` per my other comment.)

- Change the value widths to 9 too because the bytes one hits 9 digits on a fast enough machine (mine).

- Remove the spurious `)`

Reply:
* will make `>9`
* thought the `)` from how you had before had some meaning so left it in... guess not :P will remove

Reply:
Yeah, it used to be `1234 (5678/s)` that I then changed into `1234 | average: 5678/s` but missed that one `)`

Reply:
Ahh okay, got it, will remove that stray `)` then